components/script/dom/document.rs

```diff
@@ -233,6 +234,17 @@ enum FocusTransaction {
     InTransaction(Option<Dom<Element>>),
 }
 
+/// Information about a declarative refresh
+#[derive(JSTraceable, MallocSizeOf)]
+pub enum DeclarativeRefresh {
+    PendingLoad {
+        #[no_trace]
+        url: DomRefCell<ServoUrl>,
```

Comment:
If we store the enum inside of a refcell, why do we need one here as well?

Reply:
It's not needed (nor is the one below), I removed it. (my earlier implementation did require a `DomRefCell` there, and I forgot to check if it was still needed when changing it)